mailchimp/README.md

```diff
@@ -13,36 +13,30 @@ The Mailchimp integration collects metrics from campaigns and lists, directing t
 
 ## Setup
 
-### Configuration
+### Get API credentials for Mailchimp
 
-#### Get API credentials for Mailchimp
+1. Log in to your [Mailchimp account][2]. 
+2. Check the url for the Server prefix. It is the `xxxx`part of the url(eg: `https://xxxx.admin.mailchimp.com/`).
+3. Click on the profile icon and select Profile option.
+4. Navigate to the **Extras** tab and click on **API keys** from the Dropdown.
+5. Scroll down to the **Your API Keys** section and click **Create A Key**.
+6. Enter your preferred name for the API key and click on **Generate Key**. Your API key is now generated.
 
-1. Log in to your [Mailchimp account][2] and click the profile icon
-2. Navigate to the **Extras** tab
-3. Click **API keys**
-4. Scroll down to the **Your API Keys** section and click **Create A Key**
-5. Enter your preferred name for the API key and click **Ok**
-6. Once the API key is generated, copy and save it as you will only see it once
 
+### Add your Mailchimp Credentials
 
-#### Mailchimp DataDog integration configuration
+- Mailchimp Marketing API Key
+- Mailchimp Server Prefix
 
-Configure the Datadog endpoint to forward  Mailchimp metrics to Datadog.
-
-1. Navigate to the `Mailchimp` integration tile in Datadog.
-2. Add your Mailchimp credentials.
-
-| Mailchimp parameters | Description  |
```

Comment:
Can we keep the table with the parameters? It's helpful!

Reply:
@dpoint01 sure, we used to have tables, but as per the some of our earlier conversatons around README we moved from the table to the parameters list. So we feel, we should either go with any of the one i.e. table or the new list format, please let us know your thoughts?